### ISSUE TITLE
v0.11.0 breaks stories_route (?)

### ISSUE THREAD

**Issue by julianrubisch** [jonspalmer/view_component_storybook#118]
👋 

Did v0.11.0 include any breaking changes? Because after the upgrade I get this error (aborts Rails boot):

```
.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/view_component_storybook-0.11.0/lib/view_component/storybook/engine.rb:14:in `block in <class:Engine>': undefined method `stories_route' for ViewComponent::Storybook:Module (NoMethodError)
```

Since I couldn't find `stories_route` in my codebase, I think this bug probably stems from the gem itself.

Apologies for a very coarse bug report, but I thought this will probably affect a lot of other apps as well. Sorry I can't provide a PR to fix it as I'm not familiar with your codebase.

**Comment by jonspalmer** [comment]
Apologies, not enough testing of the now legacy approach of requiring this library. Should be fixed in v0.11.1

**Comment by jonspalmer** [comment]
Thanks for the quick bug report.
